align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml

```diff
@@ -9,8 +9,9 @@ inference_kwargs:
     leave_one_out_strategy: null
     normalization: null
     sort_actions: true
+    most_similar_first: false
     datasets:
       QualityOfLife: /data/shared/samba/phase1_icl/qol_all_20241106.json
       PerceivedQuantityOfLivesSaved: /data/shared/samba/phase1_icl/vol_all_20241106.json
-      Moral judgement: /data/shared/samba/phase1_icl/moral_judgement_20241106.json
+      Moral judgement: /data/shared/samba/phase1_icl/moral_judgement_20241112.json
```

Comment:
Did you regenerate this?

Reply:
Yes with the changes from the train files ST made today

Reply:
Moral judgment is ADEPT though?

Reply:
Okay that's been corrected